clients/apps/web/src/components/Profile/CreatorCard.tsx

```diff
@@ -0,0 +1,120 @@
+import { useSortable } from '@dnd-kit/sortable'
+import { CSS } from '@dnd-kit/utilities'
+import { DragIndicatorOutlined, GitHub } from '@mui/icons-material'
+import Link from 'next/link'
+import Avatar from 'polarkit/components/ui/atoms/avatar'
+import Button from 'polarkit/components/ui/atoms/button'
+import {
+  Card,
+  CardContent,
+  CardFooter,
+  CardHeader,
+} from 'polarkit/components/ui/atoms/card'
+import { useGetOrganization } from 'polarkit/hooks'
+import { organizationPageLink } from 'polarkit/utils/nav'
+import { twMerge } from 'tailwind-merge'
+
+export const CreatorCard = ({
+  organizationId,
+  disabled,
+  sortable,
+}: {
+  organizationId: string
+  disabled?: boolean
+  sortable?: ReturnType<typeof useSortable>
+}) => {
+  const organization = useGetOrganization(organizationId).data
```

Comment:
We should load this data on the server, and make sure that this feature also is server side rendered (like the rest of this page).